### PR TITLE
Fixes #39349 - 'file' as valid ParameterType

### LINKED ISSUE #39349: swagger-schema-official BaseSchema.Type : ParameterType should include 'file'
Should include 'file':

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbc8a1ec78414e0131adddd72fb44574358d156e/types/swagger-schema-official/index.d.ts#L44

because:

https://swagger.io/docs/specification/2-0/describing-responses/

@mohsen1 @bsouthga @nimerritt

### PR DESCRIPTION
**Pull request by JasonKleban**
Fixes #39349

### CODE REVIEW

File: types/swagger-schema-official/index.d.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export interface Header extends BaseSchema {
 
 // ----------------------------- Parameter -----------------------------------
 
-export type ParameterType = 'string' | 'number' | 'integer' | 'boolean' | 'array' | 'object';
+export type ParameterType = 'string' | 'number' | 'integer' | 'boolean' | 'array' | 'object' | 'file';
```
Comment (XVincentX) [inline types/swagger-schema-official/index.d.ts]: Why is this required? This is a BaseType and we're already handling the exception in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/swagger-schema-official/index.d.ts#L103

**Review by XVincentX — CHANGES_REQUESTED** [review]

File: types/swagger-schema-official/index.d.ts (hunk 1 above)
Comment (JasonKleban) [reply types/swagger-schema-official/index.d.ts]: Because `Response.schema : Schema : BaseSchema` and therefore `Response.schema.type: ParameterType`.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbc8a1ec78414e0131adddd72fb44574358d156e/types/swagger-schema-official/index.d.ts#L147

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbc8a1ec78414e0131adddd72fb44574358d156e/types/swagger-schema-official/index.d.ts#L154

And from https://swagger.io/docs/specification/2-0/describing-responses/ :

> ### Response Body
> The schema keyword is used to describe the response body. A schema can define:
> - object or array – typically used with JSON and XML APIs,
> - a primitive such as a number or string – used for plain text responses,
> - file ...

**Comment by JasonKleban** [comment]
Did you see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39352#discussion_r338590173 ?

**Comment by XVincentX** [comment]
@JasonKleban I'm sorry I haven't been able to jump on this in a timely manner. Can you reopen it?
